hasher-matcher-actioner/webapp/src/utils/constants.jsx

```diff
@@ -1,5 +1,9 @@
 /**
  * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * TODO: typescript supports enums. However, all files that use enums from here
+ * need to be converted to typescript before we can change this file into
+ * typescript.
```

Comment:
Seems like a good time to typescript these files? Can you add that in a new commit?

Reply:
From my past experiences, hard/impossible to `import <typescript module>` in a js file. 

These leaf files (ones that do not import anything else) must be the last ones to be migrated. That includes constants.jsx.

For the others, yeah, sure I can give that a shot.

Reply:
Works when I do it lol :D

1. Change file name to `constants.tsx`
2. Find and replace imports like: `s/constants';/constants.tsx';`